phpunit.xml.dist

```diff
@@ -24,4 +24,7 @@
             </exclude>
         </whitelist>
     </filter>
+    <php>
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak" />
```

Comment:
Maybe forcing Symfony 3.0 to use the latest Twig in this case avoiding deprecation warnings?